src/components/Change.vue

```diff
@@ -1,45 +1,45 @@
 <template>
   <div class="change-modal">
     <div class="modal-header">
-      <button
+      <!-- <button
```

Comment:
Worth just to delete no longer used code? You can go back in history with git if you ever need it.

Reply:
Done